README.md

```diff
@@ -40,27 +40,31 @@ For more details, see the documentation of the underlying [psutil library](https
 - `-t, --total`: Display disk usage as used/total in GB instead of a percentage.
 - `-f, --free`: Display free disk space in GB.
 
+### `#{battery}` Placeholder
+- `-t, --time`: Display the remaining battery life time.
+- `-p, --percentage`: Display the remaining battery percentage.
+- `-l, --long`: Display the remaining battery as a sentence.
+- `-c, --compact`: Display the remaining battery using and icon.
```

Comment:
```suggestion
- `-i, --icon`: Display the remaining battery using and icon.
```

It should of course be "an" and not "and", but should this argument be renamed to "icon" perhaps? Thoughts?